btk/draw_blends.py

```diff
@@ -130,6 +130,7 @@ def __init__(
         indexes=None,
         channels_last=False,
         save_path=None,
+        rng=None,
```

Comment:
here should be `seed` and not `rng`

Reply:
Why ? It is not necessary a seed, it can directly be a `numpy.random.Generator`.